tests/testthat/_snaps/type.md

```diff
@@ -0,0 +1,151 @@
+# type style
+
+    Code
+      cli_text("{.type {mtcars}}")
+    Message
+      a <data.frame> object
```

Comment:
Should we special case data frames and tibbles?

Reply:
Maybe. Although for tibbles it is good to know what kind of tibble, and there are other data.frame based classes as well, where it would be good to show the extra classes.

Reply:
Maybe special case `identical(class(x), "data.frame")` as "a data frame", and `identical(class(x), c("tbl_df", "tbl", "data.frame"))` as "a tibble"?